### DISCUSSION TITLE
Welcome to plotly.java Discussions!

### DISCUSSION THREAD

**Discussion by lwhite1** [jtablesaw/plotly.java#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlfuraydiO** [comment]
Hello, If I were to submit a PR. Should I submit it here or in tablesaw repo. since I can see jsplot exists in both repos.
